### ISSUE TITLE
error occured in run_classifier.py when runing evaluation

### ISSUE THREAD

**Issue by wing7171** [helloeve/mre-in-one-pass#11]
Q1: I wonder if train and eval code works ok in  run_classifier.py, cuz i encountered error when evaluating which is caused by inconsistent dim of label_ids and predictions.

Q2: why dim of label_ids is [None,max_num_relations] but logits is [None, 6] ?? what does max_num_relations mean?
 Could anybody please help me of exact dim transformation in output layer of this model ? Thanks in advance!!
